react/states_and_effects/project_cv_application.md

```diff
@@ -70,7 +70,8 @@ Similar to Netlify, Vercel also offers a few ways to deploy, but we will be impo
 
 1. Yes, push your React application to GitHub.
 1. Create a new Cloudflare account and log into it.
-1. Under "Account Home", select "Workers & Pages" > "Pages" > "Connect to Git".
+1. At the top of the page click "+ Add" > "Pages".
```

Comment:
```suggestion
1. At the top of the page click the blue button "+" or "Add" and navigate to "Pages" in the dropdown.
```

I did not fully understand why you made it "+ Add" but I see now why you did so. I'd suggest clarifying this a bit. What do you think?

![image](https://github.com/user-attachments/assets/01f5f796-31f7-4a9b-8dcb-b1f36d1a27ae)
![image](https://github.com/user-attachments/assets/6d8e41c0-b46e-4762-bc8d-ea9f316b080f)